File: notesapp/app/src/main/java/com/example/notesapp/MainActivity.java
package com.example.notesapp;


import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.fragment_container, new NoteListFragment())
                    .commit();
        }
    }

    public void openNoteDetail(Note note) {
        if (getResources().getBoolean(R.bool.is_landscape)) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.detail_container, NoteDetailFragment.newInstance(note))
                    .commit();
        } else {
            Intent intent = new Intent(this, NoteDetailActivity.class);
            intent.putExtra("title", note.getTitle());
            intent.putExtra("desc", note.getDescription());
            intent.putExtra("date", note.getDate());
            startActivity(intent);
        }
    }
}

File: app/src/main/java/com/example/hw1/MainActivity.kt
package com.example.hw1

import android.content.Intent
import android.os.Bundle
import android.widget.*
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val editText = findViewById<EditText>(R.id.editText)
        val button = findViewById<Button>(R.id.button)
        val switch1 = findViewById<Switch>(R.id.switch1)
        val checkBox = findViewById<CheckBox>(R.id.checkBox)
        val toggleButton = findViewById<ToggleButton>(R.id.toggleButton)
        val calendarView = findViewById<CalendarView>(R.id.calendarView)

        button.setOnClickListener {
            val text = editText.text.toString()
            Toast.makeText(this, "Вы ввели: $text", Toast.LENGTH_SHORT).show()
        }

        switch1.setOnCheckedChangeListener { _, isChecked ->
            Toast.makeText(this, if (isChecked) "Switch Включен" else "Switch Выключен", Toast.LENGTH_SHORT).show()
        }

        checkBox.setOnCheckedChangeListener { _, isChecked ->
            Toast.makeText(this, if (isChecked) "Чекбокс выбран" else "Чекбокс снят", Toast.LENGTH_SHORT).show()
        }

        toggleButton.setOnCheckedChangeListener { _, isChecked ->
            Toast.makeText(this, if (isChecked) "Toggle Включен" else "Toggle Выключен", Toast.LENGTH_SHORT).show()
        }

        calendarView.setOnDateChangeListener { _, year, month, dayOfMonth ->
            Toast.makeText(this, "Дата: $dayOfMonth.${month + 1}.$year", Toast.LENGTH_SHORT).show()
        }

        button.setOnLongClickListener {
            startActivity(Intent(this, SecondActivity::class.java))
            true
        }
    }
}
